qa/workunits/ceph-helpers.sh

```diff
@@ -146,6 +146,27 @@ function test_teardown() {
 #######################################################################
 
 ##
+# Kills a single daemon. This is an helper function fo kill_daemons
```

Comment:
Could you please document it as a first class citizen ? All functions in this file are helpers and they should all be individually tested and documented. The general idea is that someone interested in a given function does not need to look beyond the part of the file where it's defined, documented and tested. That locality is very valuable for the casual user.